src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -1612,7 +1673,7 @@ void Graph::EnforceBF16() {
     // starting from output nodes
     for (const auto& entry : outputNodesMap) {
         const auto& node = entry.second;
-        if (node->getOriginalInputPrecisionAtPort(0) == Precision::BF16)
+        if (node->getOriginalInputPrecisionAtPort(0) == inferPrec)
```

Comment:
Please fix comment above this code block (`Skip BF16 enforcement for tail`)

Reply:
OK